### PR TITLE
set custom response header.

### LINKED ISSUE #381: Getting file resource does not have content-length header
A file resource URL e.g. `https://cube.chrisproject.org/api/v1/uploadedfiles/10990/welcome.txt` does not have a `Content-Length` header.

### PR DESCRIPTION
**Pull request by Prakashh21**
Fixes #381 

**Screenshots**

Screenshot_2022-04-25_15-38-05

### CODE REVIEW

**Comment by Prakashh21** [comment]
I still have some doubts regarding this , anyways it's not making any extra api calls to the swift container.

File: chris_backend/uploadedfiles/views.py [hunk 1]
```diff
@@ -130,4 +130,6 @@ def get(self, request, *args, **kwargs):
         Overriden to be able to make a GET request to an actual file resource.
         """
         user_file = self.get_object()
-        return FileResponse(user_file.fname)
+        response = FileResponse(user_file.fname)
+        response["Content-Length"] = len(user_file.fname)
```
Comment (jennydaman) [inline chris_backend/uploadedfiles/views.py:134]: I think it would be more clear if you directly called `user_file.fname.size`

**Review by jennydaman — CHANGES_REQUESTED** [review]
There are several file APIs, `api/v1/files/`, `api/v1/uploadedfiles/`, `api/v1/pacsfiles/`, would you apply the fix to all of them? Maybe a mixin would be appropriate here...

File: chris_backend/uploadedfiles/views.py (hunk 1 above)
Comment (Prakashh21) [reply chris_backend/uploadedfiles/views.py:134]: Ok @jennydaman .

**Pull request closed without merging**